### PR TITLE
Update README.md

### LINKED ISSUE #100: Create video and add to GitHub ReadMe
A product video will be made and added to Readme.md
- [x] Script
- [x] Editing

### PR DESCRIPTION
**Pull request by himankmeattle**
README Updates
Resolves #100

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,70 @@
...
-## What you are going to make?
-- Tank - Total Annihilation Not Komplete
-- It will give users of the Android app the possibility of detecting objects by looking at a camera feed. This camera feed is recieved from the bot they control in SMCE
-- This program will let users control a smart bot in SMCE with the objective of crashing into other SMCE bots.
-- It will possibly be able to do automated bot hunting.  
+## Table of Contents
+- [Demo Video](https://github.com/DIT112-V21/group-10/tree/readme#demo-video-must-watch)
+- [What did we make?](https://github.com/DIT112-V21/group-10/tree/readme#what-did-we-make)
+- [Why did we make this? What problem was solved?](https://github.com/DIT112-V21/group-10/tree/readme#why-did-we-make-this-what-problem-was-solved)
+- [How did we make this? What kind of technology and resources we used?](https://github.com/DIT112-V21/group-10/tree/readme#how-did-we-make-this-what-kind-of-technology-and-resources-did-we-use)
```
Comment (Irina0904) [inline README.md]: *have we used

File: README.md [hunk 2]
```diff
@@ -1,20 +1,70 @@
...
+Tutorial can be found **[here.](https://github.com/DIT112-V21/group-10/blob/readme/androidGeoBot/app/src/main/assets/tank_tutorial_white.pdf)**
+
+## "Get Started" Guide for Developers
+1. Download and install [SMCE](https://github.com/ItJustWorksTM/smce-gd/releases), and  [Android Studio](https://developer.android.com/studio/), following the [setup instructions](https://github.com/ItJustWorksTM/smce-gd/wiki) for SMCE
+2. Download/clone remote repository from GitHub
+3. Go through following README.md to know how to prepare development environment for respective part
+- [Android README](https://github.com/DIT112-V21/group-10/blob/master/androidGeoBot/README.md)
+- [Arduino README](https://github.com/DIT112-V21/group-10/blob/master/arduino/README.md)
+4. Open smartcar.ino in Arduino IDE and SMCE
+5. Open othercars.ino Arduino IDE and SMCE
```
Comment (Irina0904) [inline README.md]: The developer also needs to compile both files.

File: arduino/README.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
-How to control the car:
-
-- To go forward or backward with a desired speed, you should insert "m" and then your desired speed
-  into the serial and then press Enter. If the speed input is negative the car will go backward,
-  otherwise it moves forward.
-- To change the direction of the car to a desired angle you should insert "t" and then your desired angle
-  into the serial and then press Enter. If the degree input is negative the car will turn clockwise,
-  otherwise it turns counter clockwise.
\ No newline at end of file
+To begin controlling the car you should SMCE installed. Open the program, compile the smartcar sketch. Afterwards, compile and start the othercars sketch, running as many cars you would like to be able to crash into. Finally, you should start the smartcar sketch.
```
Comment (Irina0904) [inline arduino/README.md]: *have SMCE installed

File: arduino/README.md [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
+
+The smartcar sketch is what is used for allowing the player to control the car. 
+
+The othercars sketch is used for automatic driving. The car will avoid obstacles to make it challenging for the player to crash into.
```
Comment (Irina0904) [inline arduino/README.md:7]: Is the car still avoiding obstacles? I think we changed that.

File: arduino/README.md [hunk 5]
```diff
@@ -1,15 +1,15 @@
...
+
+How to control the smartcar:
+
+- You must run the android application, and be connected to a broker, follow instructions in the android folder for set up.
```
Comment (Irina0904) [inline arduino/README.md:11]: *setup

File: arduino/README.md [hunk 6]
```diff
@@ -1,15 +1,15 @@
...
+
+- To move the car, you should move the joystick in whichever direction you would like to go.
```
Comment (Irina0904) [inline arduino/README.md]: I think it should be "move the inner circle of the joystick", just so that it's clear what we mean.

File: README.md [hunk 7]
```diff
@@ -1,20 +1,70 @@
...
+- Discord
+- Zoom
+
+## Game Tutorial for Users
+Tutorial can be found **[here.](https://github.com/DIT112-V21/group-10/blob/readme/androidGeoBot/app/src/main/assets/tank_tutorial_white.pdf)**
+
+## "Get Started" Guide for Developers
+1. Download and install [SMCE](https://github.com/ItJustWorksTM/smce-gd/releases), and  [Android Studio](https://developer.android.com/studio/), following the [setup instructions](https://github.com/ItJustWorksTM/smce-gd/wiki) for SMCE
+2. Download/clone remote repository from GitHub
+3. Go through following README.md to know how to prepare development environment for respective part
```
Comment (Irina0904) [inline README.md:74]: the following

File: README.md [hunk 8]
```diff
@@ -1,20 +1,70 @@
...
-- It can be used to cause mayhem and destruction
+## Demo Video (must watch)
 
-## How are you going to make it? What kind of technology are you going to use?
-We will use OpenCV in combination with an Android app, in order to detect objects in the SMCE world. Technologies we will use:
-- C++
-- Python
-- OpenCV
-- Android/Java
+TODO: add video link
```
Comment (Irina0904) [inline README.md]: We already have a video link, right? Maybe we should take that away now?

**Review by Irina0904 — COMMENTED** [review]
The README looks good. I've only found some minor parts that can be improved. Good job!

File: arduino/README.md (hunk 4 above)
Comment (hannahshiels) [reply arduino/README.md:7]: The othercars sketch avoids obstacles #75 .

**Review by Spidergirl93 — APPROVED** [review]
Everything looks fine to me.

**Pull request merged**